src/frontend/packages/eslint-config-people/next.js

```diff
@@ -0,0 +1,125 @@
+module.exports = {
```

Comment:
I need 2 different because of Jest stepping on Playwright tests, but yes we can DRY a bit:
https://github.com/numerique-gouv/people/pull/14/commits/86d88034ed978f121b212768a75b80f6353f6f48